src/neo4j/_async/work/session.py

```diff
@@ -180,14 +180,16 @@ async def _result_error(self, error):
 
     async def _get_server_info(self):
         assert not self._connection
-        await self._connect(READ_ACCESS, liveness_check_timeout=0)
+        await self._connect(
+            READ_ACCESS, liveness_check_timeout=0, unprepared=True
+        )
         server_info = self._connection.server_info
         await self._disconnect()
         return server_info
 
     async def _verify_authentication(self):
         assert not self._connection
-        await self._connect(READ_ACCESS, force_auth=True)
+        await self._connect(READ_ACCESS, force_auth=True, unprepared=True)
```

Comment:
```suggestion
        await self._connect(READ_ACCESS, force_auth=True, unprepared=True)
```
Very ready to be wrong here, but does setting `unprepared `to True here achieve what we need? Since `force_auth `is also True it looks to me like the conditional where `unprepared `is checked would still pass?

Reply:
Covered in a huddle, adding answer here for future reference: the `unprepared` here does not alter current behavior, but makes sure that potential future bits of pipelined work do not cause errors given that they respect `unprepared`